modules/images/dominant-color/load.php

```diff
@@ -405,3 +405,1418 @@ function dominant_color_rgb_to_hex( $red, $green, $blue ) {
 
 	return sprintf( '%02x%02x%02x', $red, $green, $blue );
 }
+
+/**
+ * Integrate with the media library.
+ */
+function dominant_color_add_media_library_integration() {
+	remove_action( 'admin_footer', 'wp_print_media_templates' );
```

Comment:
this removal isn't working here and probably needs to run earlier or later.

Reply:
thanks